### PR TITLE
nova-operator: add pre-commit tests

### PR DESCRIPTION
**Pull request by Sandeepyadav93**

### CODE REVIEW

**Review by gibizer — COMMENTED** [review]
Looks good to me

File: ci-operator/config/openstack-k8s-operators/nova-operator/openstack-k8s-operators-nova-operator-master.yaml [hunk 1]
```diff
@@ -43,6 +51,16 @@ tests:
   commands: make gofmt || true
   container:
     from: src
+- as: precommit-check
+  commands: |
+    export SKIP=no-commit-to-branch
+    export GOFLAGS=
+    export PRE_COMMIT_HOME=/tmp
```
Comment (gibizer) [inline ci-operator/config/openstack-k8s-operators/nova-operator/openstack-k8s-operators-nova-operator-master.yaml:58]: I think we need `export GOLANGCI_LINT_CACHE=/tmp` as well to make golangci-lint happy with in this CI container.

Comment (Sandeepyadav93) [reply ci-operator/config/openstack-k8s-operators/nova-operator/openstack-k8s-operators-nova-operator-master.yaml:58]: thanks, added

Comment (gibizer) [reply ci-operator/config/openstack-k8s-operators/nova-operator/openstack-k8s-operators-nova-operator-master.yaml:58]: Are you sure you added it? I don't see it.

**Review by gibizer — COMMENTED** [review]
Looks good to me

**Review by gibizer — APPROVED** [review]

**Review by gibizer — APPROVED** [review]
/lgtm

**Pull request merged**